axon-framework/events/event-bus-and-event-store.md

```diff
@@ -393,14 +393,30 @@ If no explicit `eventSerializer` is configured, events are serialized using the
 
 ## Distributing Events
 
-The use of an `EventStore` provides a simple way of distributing events: by sharing the data source.
-This is the approach taken by [Axon Server](../../axon-server-introduction.md), for example.
+To distribute events between applications, it is paramount to know whether the applications belong to the same [bounded context](../../architecture-overview/ddd-cqrs-concepts.md#bounded-context).
+If they do, they're allowed to understand each other's language.
+In other words, they should be free to understand the events altogether.
```

Comment:
Not sure what "altogether" really adds in this context. In my opinion, they aren't "allowed to" or "free to" understand the events, but when they're in the same bounded context, they just do.

Reply:
Hmm, I'll adjust this to some stronger wording, then.